src/table/utils/styling-utils.js

```diff
@@ -80,14 +80,14 @@ export function getHeaderStyle(layout, theme) {
 
 export function getHoverBackgroundColor(theme) {
   const hoverBackgroundColor = theme.getStyle('object', 'straightTable.content.hover', 'backgroundColor');
-  const contentBackgroundColor = theme.getStyle('object', 'straightTable.content', 'backgroundColor');
   const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
   const objectBackgroundColor = theme.getStyle('object', '', 'backgroundColor');
-  const isHoverBackgroundColorFromTheme = !(
-    hoverBackgroundColor === contentBackgroundColor ||
-    hoverBackgroundColor === tableBackgroundColor ||
-    hoverBackgroundColor === objectBackgroundColor
-  );
+  const isHoverBackgroundColorFromTheme =
+    !(hoverBackgroundColor === tableBackgroundColor || hoverBackgroundColor === objectBackgroundColor) ||
+    // hoverBackgroundColor is green
+    // tableBackgroundColor is red
+    // objectBackgroundColor is green
+    (hoverBackgroundColor !== tableBackgroundColor && hoverBackgroundColor === objectBackgroundColor);
```

Comment:
I think it is better to not make it super complicated here. Better to fix `getStyles` and accept we hava a possible corner case for now

Reply:
By the way, https://github.com/qlik-oss/nebula.js/pull/776 is ready to be reviewed.